### PR TITLE
Dox

### PR DESCRIPTION
**Pull request by heckj**
doc updates
- matching to latest python-keystoneclient
- updated to referencing 'keystone-all' for running the service
- a few misc other tweaks and removing Keystone Light from README

### CODE REVIEW

File: docs/source/configuration.rst [hunk 1]
```diff
@@ -78,6 +79,18 @@ Sample Configuration Files
...
 * ``etc/logging.conf.sample``
 
+Running Keystone
+================
+
+Running keystone is simply starting the services by using the command::
+
+    keystone-all
+
+Invoking this command starts up two wsgi.Server instances, configured by the
```
Comment (termie) [inline docs/source/configuration.rst:89]: awesomeee

File: docs/source/configuration.rst [hunk 2]
```diff
@@ -129,7 +142,9 @@ containers within Swift. A tenant can have zero or more users, Users can be asso
 keyword arguments
 
 * tenant_name
-* id (optional)
+* id (optional, defaults to generated ID)
```
Comment (termie) [inline docs/source/configuration.rst]: not even optional, not allowed

File: docs/source/developing.rst [hunk 3]
```diff
@@ -76,7 +76,7 @@ light integration testing to verify the keystone API against other projects.
 Test Structure
 --------------
 
-``./run_test.sh`` uses it's python cohort (``run_tests.py``) to iterate through the ``tests`` directory, using Nosetest to collect the tests and invoke them using an 
+``./run_test.sh`` uses it's python cohort (``run_tests.py``) to iterate through the ``tests`` directory, using Nosetest to collect the tests and invoke them using an
```
Comment (termie) [inline docs/source/developing.rst]: it's -> its

Comment (termie) [inline docs/source/developing.rst]: also this line seems longer than the others

File: docs/source/configuration.rst (hunk 2 above)
Comment (heckj) [inline docs/source/configuration.rst]: removing

**Comment by termie** [comment]
it looks like you added a bunch of docs for the arguments to keystone-manage commands, but they aren't really accurate, you can't set the id for anything in the system at the moment, nor do we have things like paging (you have some stuff with marker and offset)

**Comment by heckj** [comment]
I built those command options entirely from reading what methods are exposed by the python-maestroclient, using what it passed through as the options. Removed the ID one (from an earlier version I messed up earlier). The paging is in that one call - so I exposed it. Any suggestions on how to track what's exposed or not other than actually running each command and seeing what it might take?

File: docs/source/developing.rst (hunk 3 above)
Comment (heckj) [inline docs/source/developing.rst]: fixed both

**Comment by heckj** [comment]
ran through all the commands to verify their accuracy - updated the dox.

**Comment by termie** [comment]
one way of automatically making the tests work would be to come up with a way to use doctests, probably if we write a small fake object that can be easily imported to run them against, anyway, scanning the docs one last time (i hope :)) before merging :)

File: docs/source/configuration.rst [hunk 4]
```diff
@@ -175,10 +190,12 @@ keyword arguments
 * name
 * password
 * email
+* tenant_id (optional, defaults to None)
+* enabled (optional, defaults to True)
```
Comment (termie) [inline docs/source/configuration.rst:194]: booleans don't work yet in the command-line (well, you can't set them to false), not sure whether you want to mark that as a bug and just write the code fix, or reflect them in the docs

File: docs/source/configuration.rst [hunk 5]
```diff
@@ -188,31 +205,86 @@ example::
...
+	keystone-manage user update_email user=03c84b51574841ba9a0d8db7882ac645 email="someone@somewhere.com"
+
+
 ``user update_enabled``
 ^^^^^^^^^^^^^^^^^^^^^^^
 
 keyword arguments
 
+* user
+* enabled (True or False)
```
Comment (termie) [inline docs/source/configuration.rst:251]: ditto here, making this call kinda useless

File: docs/source/configuration.rst [hunk 6]
```diff
@@ -225,29 +297,81 @@ keyword arguments
...
+	role=3a751f78ef4c412b827540b829e2d7dd \
+	user=03c84b51574841ba9a0d8db7882ac645 \
+	tenant=20601a7f1d94447daa4dff438cb1c209
+
+``role remove_user_role``
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+keyword arguments
+
+* role
```
Comment (termie) [inline docs/source/configuration.rst:364]: this stuff all reminds me that the argument names in keystoneclient could be pleasantly renamed to be more consistent once this stuff all works

File: docs/source/configuringservices.rst [hunk 7]
```diff
@@ -42,13 +44,16 @@ Admin Token
 -----------
 
 For a default installation of Keystone, before you can use the REST API, you
-need to define an authorization token. This is configured in the keystone.conf file under the section ``[DEFAULT]``. In the sample file provided with the keystone project, the line defining this token is
+need to define an authorization token. This is configured in ``keystone.conf``
+file under the section ``[DEFAULT]``. In the sample file provided with the keystone project, the line defining this token is
```
Comment (termie) [inline docs/source/configuringservices.rst]: if we're making the lines within 80 chars (i'm for that where possible), we missed this one

File: docs/source/man/keystone-all.rst [hunk 8]
```diff
@@ -77,5 +79,5 @@ SEE ALSO
 SOURCE
 ======
 
-* Keystone is sourced in GitHub `Keystone <http://github.com/openstack/keystone>`__
+* Keystone source in managed in GitHub `Keystone <http://github.com/openstack/keystone>`__
```
Comment (termie) [inline docs/source/man/keystone-all.rst]: s/in/is/

File: tox.ini [hunk 9]
```diff
@@ -2,7 +2,7 @@
 envlist = py26,py27,pep8
 
 [testenv]
-deps = -r{toxinidir}/tools/pip-requires
+deps = -r{toxinidir}/tools/pip-requires-test
```
Comment (termie) [inline tox.ini:5]: does our test pip-requires contain the other requirements also? if not we should probably do that :/ (or make this do both)

File: docs/source/configuringservices.rst (hunk 7 above)
Comment (heckj) [inline docs/source/configuringservices.rst]: noted - updating and wrapping all remaining lines in there just to get it done.

**Pull request merged**